### PR TITLE
rework confirm sending screen for cake pay

### PR DESCRIPTION
**Pull request by Serhii-Borodenko**
reworked confirm sending screen for cake pay

### CODE REVIEW

File: lib/src/widgets/base_alert_dialog.dart [hunk 1]
```diff
@@ -11,6 +11,10 @@ class BaseAlertDialog extends StatelessWidget {
   VoidCallback get actionLeft => () {};
   VoidCallback get actionRight => () {};
   bool get barrierDismissible => true;
+  Color? get leftActionButtonTextColor => null;
+  Color? get rightActionButtonTextColor => null;
```
Comment (OmarHatem28) [inline lib/src/widgets/base_alert_dialog.dart:15]: not used

Comment (Serhii-Borodenko) [reply lib/src/widgets/base_alert_dialog.dart:15]: used [here](https://github.com/cake-tech/cake_wallet/pull/603/files#diff-4513851fdf25d22afd919464aeb9dac12860a3d38e29edd084873a1656616376R321-R323) to change default parameters

Comment (OmarHatem28) [reply lib/src/widgets/base_alert_dialog.dart:15]: I mean its value is not used in the widget (line 93 should be `rightActionButtonTextColor ?? etc..`)

Comment (Serhii-Borodenko) [reply lib/src/widgets/base_alert_dialog.dart:15]: Yes, indeed, fixed it. Also fixed the text color, button layout and visibility of the sending address.

**Review by OmarHatem28 — APPROVED** [review]

**Pull request merged**